### DISCUSSION TITLE
How does .deb version of milvus detect an IP address its listening?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#19304] [Q&A and General discussion]
How does .deb version of milvus detect an IP address its listening? 
Sometimes when I boot my machine and it changes an IP address I can't access my milvus, and I need a reboot.
What can be the case?

**Comment by Bennu-Li** [comment]
Can you access Milvus with the new IP?
If not, run `systemctl status milvus` to check if milvus is running properly.
